microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -43,42 +43,57 @@ Feedback in the **Issues** tool is provided by several sources, including the Ch
 
 1. Go to a webpage that contains issues to fix.  For example, open the [accessibility-testing demo page](https://microsoftedge.github.io/Demos/devtools-a11y-testing/) in a new tab or window.
 
-1. Open DevTools.  After a few seconds, the **Issues counter** (![Issues counter](./index-images/issues-counter-icon.png)) appears in the upper right corner of DevTools.
+1. Right-click the webpage and then select **Inspect**.  DevTools opens.
+
+<!--
+   After a few seconds, the **Issues counter** (![Issues counter](./index-images/issues-counter-icon.png)) appears in the upper right corner of DevTools.
 
 1. Refresh the page, because some issues are reported based on network requests.  Notice the updated count in the **Issues counter**.
+-->
+
+1. In the Quick View toolbar at the bottom of DevTools, select the **Issues** tab.
```

Comment:
This renders as 1 instead of 3, presumably because of the html comment before it, which makes markdown think that this is a new list.

Reply:
@captainbrosset - I tried now indenting comment, but according to VS Code .md preview feature, the next section of paragraphs vanishes (unlike a previous fix).  Will post investigation about this problem offline.

Reply:
per VSC preview, indent + concat the comment into 1 line works w/o losing content or resetting numbering.